gapic_plugin.py

```diff
@@ -99,6 +101,20 @@ def generate_get_set_injection(response, gapic_config, request, java_package):
           f.insertion_point = 'class_scope:' + package + '.' + item.name
           f.content = renderer.render(get_class_view(field)(resource, field, concrete_resource))
 
+      oneofs = [[] for _ in item.oneof_decl]
+      for field in item.field:
+        if field.HasField("oneof_index"):
+          oneofs[field.oneof_index].append(field)
+      for i, oneof in enumerate(item.oneof_decl):
+        f = response.file.add()
+        f.name = filename
+        f.insertion_point = 'builder_scope:' + package + '.' + item.name
+        f.content = renderer.render(oneof_fields.InsertBuilderOneof(item, oneof.name, oneofs[i], type_map))
+
+        f = response.file.add()
+        f.name = filename
+        f.insertion_point = 'class_scope:' + package + '.' + item.name
+        f.content = renderer.render(oneof_fields.InsertClassOneof(item, oneof.name, oneofs[i], type_map))
```

Comment:
This method is getting a little too long, and there is an "indentation cliff" in the middle of it that makes it hard to match up the blocks.